clients/imodels-client-management/src/IModelsClient.ts

```diff
@@ -23,6 +23,12 @@ export interface IModelsClientOptions {
   api?: ApiOptions;
   /** Additional headers to add to each request. See {@link HeaderFactories}. */
   headers?: HeaderFactories;
+  /**
+   * Retry policy that is used with {@link RestClient} when HTTP request sending fails.
+   * If `undefined` the default implementation is used with exponential backoff algorithm.
+   * See {@link DefaultHttpRequestRetryPolicy} and {@link ExponentialBackoffAlgorithm}.
```

Comment:
Would be nice to document what is the exact number of retries and what are the exact delays when using default configuration.